### ISSUE TITLE
ashape3d crashes on duplicated point

### ISSUE THREAD

**Issue by tsieger** [tlafarge/alphashape3d#4]
A particular data set (featuring a duplicated point, BTW) makes `ashape3d` to crash:

```
a<-rbind(
c(   0,    0,    0),
c(   0,    0,   28),
c(  41,   24,   22),
c(  41,   24,   22),
c(  -9,   28,   -5),
c( -28,    2,   18))
library(alphashape3d)
ashape3d(a,alpha=1)
```

It results in:

```
 *** caught segfault ***
address 0x5be1d580, cause 'memory not mapped'

Traceback:
 1: .C("int1", dup1, as.integer(nvt), as.integer(ned), in.edo, auxmu2,     Mu2, mu1, mu1up, PACKAGE = "alphashape3d")
 2: ashape3d(a, alpha = 1)
```

**Comment by tlafarge** [comment]
I identified the bug.
It has to do with the dup1 vector being the dimension of the unique number of vertices versus as.integer(nvt) being the number of vertices from the dataset.
The c code then can creates the seg fault when reading past the vector size.
We are currently thinking about what should the function do in term of duplicate points.

**Comment by tsieger** [comment]
Not sure why this was closed without a fix?!

**Comment by tlafarge** [comment]
It has been fixed in the last version 1.2
Source code has been released.
I'm waiting for the windows binary from CRAN to create a new release

We decide to warn the user that his data had duplicated points and to remove them in the dataset used for the alphashape computation.

**Comment by tsieger** [comment]
Thanks for the fix!

BTW, I dare to suggest using git / github features for tracking and fixing issues, e.g. to commit the fix and mention this issue in the commit message, such that the fix will get linked with the issue automatically and everybody can see what's going on (see e.g. https://help.github.com/articles/closing-issues-via-commit-messages/). 
Also, you can create releases in github easily (these are visible in the github project page just next to commits and branches), which can help you to organize versions well. You can learn more at http://r-pkgs.had.co.nz/release.html. (CRAN release can then follow the "github release").
